packages/mwp-tracking-plugin/src/util/clickReader.js

```diff
@@ -1,4 +1,7 @@
-import { parseIdCookie } from './idUtils'; // TODO: provide this info through new plugin
+import {
+	parseIdCookie,
+	getZonedDateTimeStringWithUTCOffset,
+} from './trackingUtils'; // TODO: provide this info through new plugin
```

Comment:
Not sure what this TODO means but should we backlog a ticket for this? If so could you add a ticket?

Reply:
Yeah, that's a pre-existing todo that i'll need to ask @mmcgahan about.

Reply:
👍 created ticket, feel free to link here https://meetup.atlassian.net/browse/WP-968